tcms/core/utils/__init__.py

```diff
@@ -9,10 +9,7 @@ def form_errors_to_list(form):
     Convert errors of form to list
     Use for Ajax.Request response
     """
-    errors = []
-    for key, value in form.errors.items():
-        errors.append((key, value[0]))
-    return errors
+    return [(key, value[0]) for key, value in form.errors.items()]
```

Comment:
Function form_errors_to_list refactored with the following changes:

- Convert for loop into list comprehension ([list-comprehension](https://docs.sourcery.ai/refactorings/list-comprehension/))
- Inline variable that is immediately returned ([inline-immediately-returned-variable](https://docs.sourcery.ai/refactorings/inline-immediately-returned-variable/))